src/main/java/nl/knaw/dans/sword2/core/service/DepositHandlerImpl.java

```diff
@@ -116,7 +116,7 @@ public Deposit createDepositWithPayload(String collectionId, Depositor depositor
         }
         catch (HashMismatchException | IOException | InvalidDepositException e) {
             // cleanup files
-            cleanupFile(path);
+//            cleanupFile(path);
```

Comment:
@ericdevries : why is this commented out?

Reply:
good catch, that should not have been